phd-tests/framework/src/test_vm/mod.rs

```diff
@@ -905,3 +866,54 @@ impl Drop for TestVm {
         let _ = self.cleanup_task_tx.send(task);
     }
 }
+
+/// Attempts to ensure that the Propolis server referred to by `client` is in
+/// the `Destroyed` state by stopping any VM that happens to be running in that
+/// server.
+///
+/// This function is best-effort.
+async fn try_ensure_vm_destroyed(client: &Client) {
+    match client.instance_get().send().await.map(|r| r.instance.state) {
+        Ok(InstanceState::Destroyed) => return,
+        Err(e) => warn!(?e, "error getting instance state from dropped VM"),
```

Comment:
tiny nit: the name `e` will show up in the logs, so i might change this to 
```suggestion
        Err(error) => warn!(?error, "error getting instance state from dropped VM"),
```

Reply:
Done here (and below) in 93b17b55.